### ISSUE TITLE
Set default config for namespace and author

### ISSUE THREAD

**Issue by Schrank** [composer/composer#11243]
I just created a composer.json file via `composer init` and was wondering how I can set default values for the package namespace  and the author. I read the documentation, but couldn't find anything about it. Obviously it is read somewhere, where? 😅  I would like to change it.

I see a ton of options here, but is anything of this set via global config? If yes I'm happy to add documentation, if not I'm happy to add a PR for it :-)
https://github.com/composer/composer/blob/main/src/Composer/Command/InitCommand.php#L272-L292

Same for the author:
https://github.com/composer/composer/blob/main/src/Composer/Command/InitCommand.php#L321-L337

So if I'm correct, it should be documented here:

https://getcomposer.org/doc/03-cli.md#environment-variables

and should work like the other env vars, correct? Then I would provide a PR with a couple of lines description and add `COMPOSER_DEFAULT_VENDOR` and `COMPOSER_DEFAULT_AUTHOR `.

```
-> % composer init

  Welcome to the Composer config generator

This command will guide you through creating your composer.json config.

Package name (<vendor>/<name>) [fabian/shopify-tools]: winkelwagen/shopify-tools
Description []:
Author [Fabian Blechschmidt <blechschmidt@fabian-blechschmidt.de>, n to skip]:
```

**Comment by Seldaek** [comment]
Yes a PR for the docs would be great. I'd add these to 03-cli and ref them in the `init` docs as it is really specific to that command.

**Comment by IonBazan** [comment]
How about having a global config which can store pretty much any default content for `composer.json`, including `require` and `require-dev` too. This would serve as a template for `init` runs.

```json
{
    "config": {
        "defaults": {
            "namespace": "MyApp\\",
            "vendor": "my-vendor",
            "authors": [
                {
                    "name": "Your Name",
                    "email": "test@example.com"
                }
            ],
            "license": "MIT",
            "require-dev": {
                "phpunit/phpunit": "^9.5"
            }
        }
    }
}

```

**Comment by Seldaek** [comment]
Yeah I guess this would be ok, tho maybe would call this `init-defaults`. But do people really create so many projects that they need a template for their composer.json? :s
